src/components/basic/ProgressTabs.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+const ProgressTabs = (props) => {
+  const { className, tabItems = [] } = props;
+  const cssClasses = classnames("dg_progress-tabs", className);
+
+  return (
+    <div>
+      <ol className={cssClasses}>
+        {tabItems.map((item) => {
```

Comment:
So in general from what i'm seeing the react community they recommend taking this mapped item and making it it's own component so can we create `ProgressTabsItem` the li?

Reply:
broke it out